CHANGELOG.md

```diff
@@ -1,3 +1,147 @@
+<a name="2.0.0-beta.11"></a>
+# [2.0.0-beta.11 carapace-parapet](https://github.com/angular/material2/compare/2.0.0-beta.10...2.0.0-beta.11) (2017-09-21)
+
+
+### Highlights
+* Each `@angular/material` component is now bundled into its own javascript file. This will allow
+tools like webpack to more easily load _only_ the components being used in an application.
+* New stepper component! The base behavior lives in `@angular/cdk` with Material Design flavors in
+`@angular/material`.
+
+
+### Breaking changes
+
+* Angular Material now requires **Angular 4.4.3 or greater**
+* MaterialModule has been removed. (#6803) ([cf1ece0](https://github.com/angular/material2/commit/cf1ece0))
```

Comment:
Nit: `MaterialModule` instead of MaterialModule. Put deprecation notice note in front of the PR # instead of behind? Easier to see.